### ISSUE TITLE
Will Docker images be updated?

### ISSUE THREAD

**Issue by ameinild** [butlerx/wetty#481]
With a new release (2.6.0) will the Docker images be updated?

And will the "easy bugs" (some of which I have reported, and also have a fix ready for) be included in a future update?

Thanks.

**Comment by l33tlamer** [comment]
Just to chime in on this, now months later and at version 2.7.0, the Docker images are still not updated? Are there any plans for this? It looks like the Dockerfile has been updated a while ago already.

**Comment by butlerx** [comment]
The issue is that the action runner is timing out building the images and pushing them. when I look at docker hub it tells me images are pushed but not publically I've no idea why this is the case.

**Comment by ameinild** [comment]
Yeah it's strange because Docker hub says "Updated 8 hours ago", but no updates are visible. Did any permissions change recently? I know Github introduced 2FA, but I guess this shouldn't have any impact on this.. 🤔

**Comment by l33tlamer** [comment]
Would be a option to push the images to Githubs package repo instead? Atleast for testing to see if the issue is entirely related to Docker Hub repo or not.

**Comment by ameinild** [comment]
> Stale issue message

Maybe, but the latest images are still not public, for whatever reason...

**Comment by Sinderan** [comment]
Is there any update on this? The newest image on DockerHub is 2.5.0

**Comment by arialhamed** [comment]
> The issue is that the action runner is timing out building the images and pushing them. when I look at docker hub it tells me images are pushed but not publically I've no idea why this is the case.

Regarding this, is it true that the images at Docker Hub that we're pulling are updated, just not that Docker Hub isn't saying it's updated? If it's not then that's a bit of a concern since there's been a handful of vulnerabilities (from NPM/Node.js) that needs to be updated to those images

**Comment by butlerx** [comment]
this is done with a version that has updated all its deps
